esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -955,6 +1170,112 @@ bool CWsESDLConfigEx::onConfigureESDLBindingMethod(IEspContext &context, IEspCon
                     StringBuffer msg;
                     publishESDLMethod(espProcName.str(), espBindingName.str(), esdlDefIdSTR.toLowerCase().str(), methodName, &item, override, msg);
                     resp.updateStatus().setDescription(msg.str());
+
+                    double ver = context.getClientVersion();
```

Comment:
after configuring a method within a binding (ESDLBinding/Definition/Methods/Method) fetch the entire binding, and report it in the response